code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -73,7 +73,7 @@
 // Use this when setting the ai_eye's location.
 // It will also stream the chunk that the new loc is in.
 
-/mob/camera/ai_eye/proc/setLoc(destination, force_update = FALSE)
+/mob/camera/ai_eye/proc/setLoc(destination, force_update = FALSE, dir) //NSV13 - AI Custom Holographic Form
```

Comment:
I'm surprised the linter didn't complain about this. Usually unnamed parameters are required to come before named ones.

Reply:
They're all named because it's a proc definition, there's no restrictions on the position of predefined entries in a definition though.